src/Symfony/Component/Finder/Glob.php

```diff
@@ -42,10 +42,11 @@ class Glob
      * @param bool   $strictLeadingDot
      * @param bool   $strictWildcardSlash
      * @param string $delimiter           Optional delimiter
+     * @param bool   $caseSensitive       Add case-sensitive modifier to regex
      *
      * @return string regex The regexp
      */
-    public static function toRegex($glob, $strictLeadingDot = true, $strictWildcardSlash = true, $delimiter = '#')
+    public static function toRegex($glob, $strictLeadingDot = true, $strictWildcardSlash = true, $delimiter = '#', bool $caseSensitive = true)
```

Comment:
we cannot add the argument here as that is a BC break for any class extending this one